app/uk/gov/hmrc/helplinefrontend/monitoring/auditing/DeviceIdService.scala

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I can't see where this file is used?

Reply:
File has been removed. 
Thought it was used to get the device-id but there already are functions to get that information.